test-tools/wamr-ide/VSCode-Extension/.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Same above

Reply:
This file is to format the code and it is not necessary if the developers don't use `prettier`, will be removed